src/test/java/org/killbill/billing/plugin/util/http/TestHttpClient.java

```diff
@@ -75,6 +77,33 @@ public void testPOSTRequest() throws Exception {
         Assert.assertEquals(newCar.type, car.type);
     }
 
+    @Test(groups = "slow") // Could be fast but we've never mixed groups historically in the same file
```

Comment:
Nit: You could create another test file

Reply:
It's quite convenient to have a single test file per class (helps jumping to the test methods in IntelliJ).